### PR TITLE
Allow creating private organizations

### PR DESCRIPTION
**Pull request by hieplpvip**

### CODE REVIEW

File: judge/models/profile.py [hunk 1]
```diff
@@ -112,6 +112,8 @@ class Meta:
         permissions = (
             ('organization_admin', _('Administer organizations')),
             ('edit_all_organization', _('Edit all organizations')),
+            ('create_private_organization', _('Create private organizations')),
```
Comment (leduythuccs) [inline judge/models/profile.py:115]: Why do you need this perm? I think the default `add_organization` is enough.

**Review by leduythuccs — CHANGES_REQUESTED** [review]

File: judge/forms.py [hunk 2]
```diff
@@ -309,14 +309,22 @@ def get_choices():
...
-        fields = ['name', 'is_open', 'is_unlisted', 'about', 'logo_override_image', 'admins']
+        fields = ['name', 'slug', 'is_open', 'is_unlisted', 'about', 'logo_override_image', 'admins']
         widgets = {'admins': Select2MultipleWidget(attrs={'style': 'width: 200px'})}
         if HeavyPreviewPageDownWidget is not None:
             widgets['about'] = HeavyPreviewPageDownWidget(preview=reverse_lazy('organization_preview'))
 
+    def __init__(self, *args, **kwargs):
+        user = kwargs.pop('user', None)
+        super(OrganizationForm, self).__init__(*args, **kwargs)
+        if not user.has_perm('judge.change_open_organization'):
```
Comment (leduythuccs) [inline judge/forms.py:323]: `if user and not user.has_perm...`

**Review by leduythuccs — APPROVED** [review]
LGTM

**Comment by phanvchung** [comment]
hợp nhất cam kết [ad0a13b](https://github.com/VNOI-Admin/OJ/commit/ad0a13b71658403fe33e577179d58ac2ad4b4857) vào bậc thầy

**Pull request merged**